core/src/controls.ts

```diff
@@ -16,7 +16,39 @@
  */
 
 import * as tf from '@tensorflow/tfjs';
-import {ChordEncoder} from './chords';
+import {ChordEncoderType, chordEncoderFromType} from './chords';
+
+export type BinaryCounterArgs = {numSteps: number, numBits: number};
```

Comment:
I like how you have the spec set up, but it differs from how we're defining it in the DataConverter. Can you make the two consistent however you prefer?

Reply:
Done.